yaksh/test_models.py

```diff
@@ -204,6 +207,29 @@ def test_get_status(self):
         # Then
         self.assertEqual(status, module_status)
 
+        # Module in progress
+
+        # Given
+        self.course_status.completed_units.add(self.learning_unit_one)
```

Comment:
Making a change on a object created during setup can cause a lot of problems in further testcases. Please avoid to add/delete/update class objects in a class method or atleast revert them back at the end of the test case.

Reply:
resolved